README.md

```diff
@@ -1,24 +1,70 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## Ruby version
 
-Things you may want to cover:
+see [.ruby-version](.ruby-version)
 
-* Ruby version
+## Database creation
 
-* System dependencies
+with postgres running:
+`bundle exec rails db:create`
 
-* Configuration
+## Database initialization
 
-* Database creation
+`bundle exec rails db:create`
 
-* Database initialization
+## How to run the test suite
 
-* How to run the test suite
+`bundle exec rspec`
 
-* Services (job queues, cache servers, search engines, etc.)
+## Run the server
 
-* Deployment instructions
+`bundle exec rails s`
 
-* ...
+or with docker:
+
+`docker-compose up`
+
+all previous comands can be run inside the server container with `docker-compose exec app bash`
+
+## Endpoints
+
+### traces
+
+`POST /traces`
+
+expects the following json structure:
+
+```json
+{
+  service: "test_service",
+  service_version: "0.0.1",
+  request: { ... }, # any json object
+  response: { ... }, # any json object
+  request_ts: 123456789.5, # unixtimestamp in seconds, can be in microsecond accuracy
+  response_ts: 123456791.5 # unixtimestamp in seconds, can be in microsecond accuracy
+}
+```
```

Comment:
```suggestion
  "service": "test_service",
  "service_version": "0.0.1",
  "request": { "any": "json object" },
  "response": { "any": "json object" },
  "request_ts": 123456789.5, # unixtimestamp in seconds, can be in microsecond accuracy
  "response_ts": 123456791.5 # unixtimestamp in seconds, can be in microsecond accuracy
}
```
Where `request_ts` and `response_ts` are Unix timestamps in seconds (microsecond accuracy is possible)
```

Reply:
Oh forgot to delete the # comments on lines 44, 45

Reply:
I would make it valid JSON because otherwise GitHub will annoy us with those red error highlights